viewing_party/party.py

```diff
@@ -1,23 +1,169 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    if not title or not genre or not rating: 
+        return None
+    
+    new_movie = {"title": title, "genre": genre, "rating": rating}
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+
+    if not user_data or not movie: 
+        return None
+    
+    user_data["watched"].append(movie)
+    return user_data

```

Comment:
Something that you will see often is spaces between lines of code like these. More generally, you will see blocks of code that serve different purposes surrounded by whitespace. This visually signals that these blocks serve different purposes. For example, 
a block handling the creation of a frequency map and a block looping through a list to fill said dictionary would have a space between them. 
```suggestion
    user_data["watched"].append(movie)
    
    return user_data

```